kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/group/OffsetAcker.java

```diff
@@ -14,48 +14,34 @@
 package io.streamnative.pulsar.handlers.kop.coordinator.group;
 
 import io.streamnative.pulsar.handlers.kop.offset.OffsetAndMetadata;
-import io.streamnative.pulsar.handlers.kop.utils.KopTopic;
 import io.streamnative.pulsar.handlers.kop.utils.MessageIdUtils;
-import java.io.Closeable;
 import java.nio.ByteBuffer;
 import java.util.Map;
-import java.util.Set;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ConcurrentHashMap;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.kafka.clients.consumer.internals.ConsumerProtocol;
 import org.apache.kafka.clients.consumer.internals.PartitionAssignor.Assignment;
 import org.apache.kafka.common.TopicPartition;
-import org.apache.pulsar.client.api.Consumer;
-import org.apache.pulsar.client.api.ConsumerBuilder;
+import org.apache.pulsar.client.admin.PulsarAdmin;
 import org.apache.pulsar.client.api.MessageId;
-import org.apache.pulsar.client.api.SubscriptionInitialPosition;
-import org.apache.pulsar.client.impl.PulsarClientImpl;
 
 /**
  * This class used to track all the partition offset commit position.
  */
 @Slf4j
-public class OffsetAcker implements Closeable {
+public class OffsetAcker {
 
-    private final ConsumerBuilder<byte[]> consumerBuilder;
+    private final PulsarAdmin pulsarAdmin;
 
-    public OffsetAcker(PulsarClientImpl pulsarClient) {
-        this.consumerBuilder = pulsarClient.newConsumer()
-                .receiverQueueSize(0)
-                .subscriptionInitialPosition(SubscriptionInitialPosition.Earliest);
+    public OffsetAcker(PulsarAdmin pulsarAdmin) {
+        this.pulsarAdmin = pulsarAdmin;
     }
 
-    // map off consumser: <groupId, consumers>
-    Map<String, Map<TopicPartition, CompletableFuture<Consumer<byte[]>>>> consumers = new ConcurrentHashMap<>();
-
     public void addOffsetsTracker(String groupId, byte[] assignment) {
         ByteBuffer assignBuffer = ByteBuffer.wrap(assignment);
         Assignment assign = ConsumerProtocol.deserializeAssignment(assignBuffer);
         if (log.isDebugEnabled()) {
             log.debug(" Add offsets after sync group: {}", assign.toString());
         }
-        assign.partitions().forEach(topicPartition -> getConsumer(groupId, topicPartition));
```

Comment:
maybe here we need to init/add the original cursor to keep the backlog position.